examples/step-34/step-34.cc

```diff
@@ -320,7 +320,7 @@ namespace Step34
     , dh(tria)
     , mapping(mapping_degree, true)
     , wind(dim)
-    , singular_quadrature_order(5)
+    , singular_quadrature_order(5) // Not sure if this one needs to be dependent on the degree, too?
```

Comment:
You'll want to remove the comment here. Let's leave the value as it is right now.